src/wp-includes/formatting.php

```diff
@@ -4041,16 +4041,75 @@ function wp_trim_excerpt( $text = '', $post = null ) {
  * @param string $text      Text to trim.
  * @param int    $num_words Number of words. Default 55.
  * @param string $more      Optional. What to append if $text needs to be trimmed. Default '&hellip;'.
+ * @param bool   $allow_tags Optional. Whether to allow HTML tags in the text. Default false.
```

Comment:
We should add a `since 6.6.0` mentioning that the `$allow_tags` param was added.

Reply:
@audrasjb added `since` for `$allow_tags` param please check.